functions/go/set-namespace/main.go

```diff
@@ -1,75 +1,139 @@
+// This file will be processed and embedded to pluginator.
+
 package main
 
 import (
 	"fmt"
 	"os"
-	"strings"
 
+	"sigs.k8s.io/kustomize/api/k8sdeps/kunstruct"
+	"sigs.k8s.io/kustomize/api/resmap"
+	"sigs.k8s.io/kustomize/api/resource"
+	"sigs.k8s.io/kustomize/api/types"
 	"sigs.k8s.io/kustomize/kyaml/fn/framework"
-	"sigs.k8s.io/kustomize/kyaml/yaml"
+	"sigs.k8s.io/yaml"
 )
 
-var namespace string
-var allowClusterScoped bool
+//nolint
+func main() {
+	var plugin *plugin = &KustomizePlugin
+	defaultConfigString := `
+- path: metadata/namespace
+  create: true
+- path: subjects
+  kind: RoleBinding
```

Comment:
Why is this the default? Why RoleBinding/ClusterRolebinding?

Reply:
They are from the default config in kustomize. I will remove RoleBinding/ClusterRoleBinding